src/fparser/two/Fortran2003.py

```diff
@@ -12030,40 +12038,58 @@ def match(string):
 
 
 class Stmt_Function_Stmt(StmtBase):  # R1238
-    """
-    <stmt-function-stmt>
-    = <function-name> ( [ <dummy-arg-name-list> ] ) = Scalar_Expr
+    """Implements the matching of a Fortran statement function statement.
+
+    stmt_function_stmt is
+                  function-name ( [ dummy-arg-name-list ] ) = scalar-expr
+
     """
 
     subclass_names = []
     use_names = ["Function_Name", "Dummy_Arg_Name_List", "Scalar_Expr"]
 
     @staticmethod
     def match(string):
-        i = string.find("=")
-        if i == -1:
-            return
-        expr = string[i + 1 :].lstrip()
+        """Implements the matching for an statement function statement.
+
+        :param str string: the string to match with as a statement
+            function statement.
+
+        :returns: a tuple of size 1 containing a Stmt_Function_Stmt \
```

Comment:
This method appears to return a 3-tuple. Please could you check and update the rtype too if necessary?

Reply:
stmt_function_stmt changes should not have been in this PR. I've removed them.